### PR TITLE
Perform IP start and end range validation

### PR DESCRIPTION
**Pull request by nahuelhm17**
#### Checklist :

  - [X] I have added the relevant documentation.
  - [X] My branch is up-to-date with the upstream master branch.
  - [X] I have fully tested the changes locally.

#### Development environment

    OS: Linux
    Python Version: 3.6.5

### CODE REVIEW

File: src/vault.py [hunk 1]
```diff
@@ -39,6 +39,24 @@ def check_ip(ip: str):
         sys.exit(1)
 
 
+def check_ip_range(ip_range: int):
+    """
+    Check whether the input IP range is valid or not
+    """
+    try:
+        number = int(ip_range)
```
Comment (abhisharma404) [inline src/vault.py]: It would be better if we use a suitable variable name instead of `number`.

File: src/vault.py [hunk 2]
```diff
@@ -39,6 +39,24 @@ def check_ip(ip: str):
...
+    except ValueError:
+        colors.error('Please enter a valid number for the IP range')
+        LOGGER.error('[-] Please enter a valid number for the IP range')
+        sys.exit(1)
+    else:
+        if number > 0 and number < 255:
```
Comment (abhisharma404) [inline src/vault.py]: `ip_start_range` < `ip_end_range`, this condition is not checked anywhere.

File: src/vault.py (hunk 1 above)
Comment (nahuelhm17) [reply src/vault.py]: Any suggestion?

File: src/vault.py (hunk 2 above)
Comment (nahuelhm17) [reply src/vault.py]: Oh! I forget that. I will fix it

File: src/vault.py (hunk 1 above)
Comment (abhisharma404) [reply src/vault.py]: Why not `ip_range` ?

File: src/vault.py (hunk 2 above)
Comment (nahuelhm17) [reply src/vault.py]: I think that this problem is also solved now

File: src/vault.py (hunk 1 above)
Comment (nahuelhm17) [reply src/vault.py]: I think that this problem is solved now

**Pull request merged**